src/bin/owi.ml

```diff
@@ -39,6 +39,21 @@ let files =
   let f = existing_non_dir_file in
   Cmdliner.Arg.(value & pos_all f [] (info [] ~doc))
 
+let sourcefile ty =
+  let doc = Fmt.str "%s source file" ty in
```

Comment:
I believe using "Source file" or "Input file" is enough (some commands like `owi opt` accepts both kind of file).